cmd/chainbridge/flags.go

```diff
@@ -25,6 +25,11 @@ var (
 		Usage: "Path to keystore directory",
 		Value: DefaultKeystorePath,
 	}
+
+	HTTPConnectionFlag = cli.BoolFlag{
+		Name:  "http",
+		Usage: "Enables RPC HTTP Client",
+	}
```

Comment:
As this only applies to ethereum, this should be part of ethereum `opts` (like `gasPrice`) rather than a global flag.